src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -103,148 +108,160 @@ public void start(Promise<Void> startFuture) {
 
     callAdminHealthCheckService();
 
-    // initialize the TokenCache
+    // Initialize the TokenCache
     TokenCacheFactory.initialize(config()
         .getInteger(SYS_TOKEN_CACHE_CAPACITY, DEFAULT_TOKEN_CACHE_CAPACITY));
 
-
-    // We need to reduce the complexity of this method...
     setupHanlders();
 
-    //set Config object's defaults
-    int port = config().getInteger("port"); // move port to netServerOptions
-    NetServerOptions options = new NetServerOptions(
-        config().getJsonObject("netServerOptions", new JsonObject()))
-        .setPort(port);
-
-    server = vertx.createNetServer(options);
-
-    log.info("Deployed verticle at port {}", port);
-
-    final Metrics metrics = Metrics.getMetrics(port);
-    metricsMap.putIfAbsent(port, metrics);
-
-    server.connectHandler(socket -> {
+    JsonArray ports = config().getJsonArray("ports");
```

Comment:
I like the simplicity of changing `port` to `ports` from an int to an array, but it is a breaking change. What if we added this new feature but didn't break existing configurations that don't make use of multiple ports? I worry that doing it as a breaking change will cause all configurations for all sip2 tenants to be upgraded. But if we did it not as a breaking change, that work wouldn't need to happen. Configs in the old and new forms would both work.

Could do it in a couple of ways. One would be to keep the `port` property as an integer, and add a new `ports` property. Another would be to keep the `port` property and just allow it to be an integer or list. This probably will require a bit more code but could probably be limited to a `getPorts` method that gives you the list and throws helpful exceptions for invalid configs like having both properties defined (in the first idea of having two properties, port and ports).

Reply:
Done, please check.